man/systemd-fsck@.service.xml

```diff
@@ -18,25 +18,28 @@
   <refnamediv>
     <refname>systemd-fsck@.service</refname>
     <refname>systemd-fsck-root.service</refname>
+    <refname>systemd-fsck-usr.service</refname>
     <refname>systemd-fsck</refname>
     <refpurpose>File system checker logic</refpurpose>
   </refnamediv>
 
   <refsynopsisdiv>
     <para><filename>systemd-fsck@.service</filename></para>
     <para><filename>systemd-fsck-root.service</filename></para>
+    <para><filename>systemd-fsck-usr.service</filename></para>
```

Comment:
hmm, this unit does not exist? there's only systemd-fsck-root.service, but no systemd-fsck-usr.service.

The only reason "systemd-fsck-root.service" is kinda special is that on systems which do not use an initrd the kernel might mount the rootfs before it is checked, hence we need a special invocation that runs after it is already mounted, for compat with old stuff.

But for /usr/ that#s not necessary, because the kernel cannot mount /usr/ on its own, it *must* be done through an initrd, and if that's the case it's the initrd's job to fsck the fs *before* mounting it.

Reply:
It exists, at least in the code base.
```
$ git grep systemd-fsck-usr
src/basic/special.h:#define SPECIAL_FSCK_USR_SERVICE "systemd-fsck-usr.service"
$ git grep SPECIAL_FSCK_USR_SERVICE
src/basic/special.h:#define SPECIAL_FSCK_USR_SERVICE "systemd-fsck-usr.service"
src/shared/generator.c:                const char *unit, /* Either SPECIAL_FSCK_ROOT_SERVICE or SPECIAL_FSCK_USR_SERVICE */
src/shared/generator.c:                        r = write_fsck_sysroot_service(SPECIAL_FSCK_USR_SERVICE, dir, what, NULL);
src/shared/generator.c:                        fsck = SPECIAL_FSCK_USR_SERVICE;
src/shared/generator.c:                fsck_unit = SPECIAL_FSCK_USR_SERVICE;
```

Reply:
And the generated unit seems to have `Documentation=man:systemd-fsck-usr.service(8)`.

Reply:
I think we should avoid units that are fully generated from code. It is rather confusing for users when the unit file cannot be found on disk. But if we have them, we should document them.